articles/GPTs for Business.md

```diff
@@ -0,0 +1,698 @@
+
+If your source system does not have these capabilities (a relevant current example that does not support these would be Notion because it does not currently have and endpoint that supports queries of the content itself vs. just the title), you will need to implement the content search locally, which we will not cover in this cookbook but do plan to cover in its own cookbook in the future. ****Some examples of systems that have APIs with the advanced search capabilities needed are:
+
+- **[Confluence’s Advanced searching using CQL](https://developer.atlassian.com/cloud/confluence/advanced-searching-using-cql/)**
+- [**Google Drive API with Search**](https://developers.google.com/drive/api/guides/search-files)
+- [**Sharepoint’s Search REST API**](https://learn.microsoft.com/en-us/sharepoint/dev/general-development/sharepoint-search-rest-api-overview)
+
+For our example, we will create a “Confluence Savant” that connects our GPT to a Product Wiki to help employees get educated answers to their product questions. 
+
+We will start by creating our Action, which will leverage an OpenAPI spec + authentication to access the relevant Confluence endpoints we need our GPT to be able to access to return relevant knowledge to answer questions. 
+
+Let’s start with the spec and then finish with the auth.
+1. **Creating the OpenAPI Spec**
+
+The easiest way to create an OpenAPI spec would be to use our native capability to import the full schema from a URL. 
+
+If we are less technically inclined, we might use ChatGPT to help us create the OpenAPI spec. For those of you who work regularly with integrations and are especially familiar with Swagger you likely can proceed to visiting the developer site, in our case: **[Confluence’s Advanced searching using CQL](https://developer.atlassian.com/cloud/confluence/advanced-searching-using-cql/)** and creating the OpenAPI spec to input yourself. 
+
+For those less technically inclined, you can simply ask ChatGPT the following:
+```markdown
+Browse this page https://developer.atlassian.com/cloud/confluence/advanced-searching-using-cql/ and create an OpenAPI spec complete with a servers section and operation ids for searching for words inside of content in Confluence.
+```
+This should return a Spec that you can copy and paste into the Schema text box. Here is our example that ChatGPT provided, shown below (note: since no limits are used, this will limit the returned knowledge to the default of 25 articles)
```

Comment:
This is the # of articles returned. I added language to clarify. (note: this is not leveraging the Confluence APIs limit parameter, so it will default the number of articles to 25 articles per call, you could always add this parameter to the spec)